include/swift/SIL/SILInstruction.h

```diff
@@ -52,6 +53,9 @@ class ValueDecl;
 class VarDecl;
 class FunctionRefInst;
 
+SILValue getUndefValue(SILType Ty);
```

Comment:
I thought we no longer needed getUndefValue. If we do, please comment it's meaning/purpose here.